doc/3d-snapshot/BF.c

```diff
@@ -2,21 +2,81 @@
 
 #include "BF.h"
 
-static inline uint64_t ValidateDummyEmp(uint64_t StartPosition)
+static inline uint64_t
+ValidateDummyEmp(
+  uint8_t *Ctxt,
+  void
+  (*Err)(
+    EverParseString x0,
+    EverParseString x1,
+    EverParseString x2,
+    uint8_t *x3,
+    uint32_t x4,
+    uint8_t *x5,
+    uint64_t x6,
+    uint64_t x7
+  ),
+  uint32_t Uu,
+  uint8_t *Input,
+  uint64_t StartPosition
+)
 /*++
     Internal helper function:
         Validator for field _dummy_emp
         of type BF._dummy
 --*/
 {
   /* Validating field emp */
-  return StartPosition;
+  uint64_t positionAfterDummy = StartPosition;
```

Comment:
@aseemr @tahina-pro : Here's an always-succeeding unit validator that ended up getting wrapped with an error handler